setup_utils/find_libfuzzer.sh

```diff
@@ -24,11 +24,11 @@ if [[ "$uname" == "Darwin" ]]; then
 elif [[ "$uname" == "Linux" ]]; then
   machine="$(uname -m)"
   if [[ "$machine" == "x86_64" ]]; then
-    libpath="lib/linux/libclang_rt.fuzzer_no_main-x86_64.a"
+    libpath="lib/x86_64-unknown-linux-gnu/libclang_rt.fuzzer_no_main.a"
```

Comment:
I frankly don't need the changes in this file for OSS-Fuzz, I can hack around this files problems with LIBFUZZER_LIB. I'm happy to undo my changes to this file and let you figure out how to handle the differences between clang versions.